### PR TITLE
[WIP] PSL Year in Review 

### PR DESCRIPTION
**Pull request by jdebacker**
This PR creates a "year in review" post for the PSL blog.

cc @MattHJensen @MaxGhenis @towashington

Please comment if I'm missing something or could better explain things.  Any suggestions welcome.

### CODE REVIEW

File: _posts/2020-12-23-2020_year_in_review.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ description: Highlights from PSL in 2020
+ categories: [PSL, PSL-Foundation]
+ author: Jason DeBacker
+ title: "2020: A Year in Review"
+---
+
+
+
+This year has been one to forget!
+But 2020 did have it's bright spots, especially in the PSL community.
```
Comment (MaxGhenis) [inline _posts/2020-12-23-2020_year_in_review.md:13]:
```suggestion
But 2020 did have its bright spots, especially in the PSL community.
```

File: _posts/2020-12-23-2020_year_in_review.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+This post reviews some of the highlights from the year.
+
+The Library was able to welcome two new models to the catalog this year: [microdf](https://github.com/PSLmodels/microdf) and [OpenFisca-UK](https://github.com/PSLmodels/openfisca-uk).
+mircodf provides a number of useful tools for use with economic survey data.
```
Comment (MaxGhenis) [inline _posts/2020-12-23-2020_year_in_review.md:17]:
```suggestion
microdf provides a number of useful tools for use with economic survey data.
```
also should we code-font the packages?

File: _posts/2020-12-23-2020_year_in_review.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+OpenFisca-UK builds off the [OpenFisca](https://openfisca.org/en/) platform, offering a microsimulation models for tax and benefits programs in the UK.
```
Comment (MaxGhenis) [inline _posts/2020-12-23-2020_year_in_review.md:18]:
```suggestion
OpenFisca-UK builds off the [OpenFisca](https://openfisca.org/en/) platform, offering a microsimulation models for tax and benefit programs in the UK.
```

File: _posts/2020-12-23-2020_year_in_review.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+In addition, four new models were added to the Library as incubating projects.  The [ui-calculator](https://github.com/PSLmodels/ui_calculator) model has recieved a lot of attention this year in the U.S., as it provides the capability to calculate unemployment insurance benefits across U.S. states, a major mode of delivering financial relief to individual during the COVID crisis.
```
Comment (MaxGhenis) [inline _posts/2020-12-23-2020_year_in_review.md:20]:
```suggestion
In addition, four new models were added to the Library as incubating projects.  The [ui-calculator](https://github.com/PSLmodels/ui_calculator) model has received a lot of attention this year in the U.S., as it provides the capability to calculate unemployment insurance benefits across U.S. states, a major mode of delivering financial relief to individuals during the COVID crisis. 
```

File: _posts/2020-12-23-2020_year_in_review.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+This year has been one to forget!
+But 2020 did have it's bright spots, especially in the PSL community.
+This post reviews some of the highlights from the year.
+
+The Library was able to welcome two new models to the catalog this year: [microdf](https://github.com/PSLmodels/microdf) and [OpenFisca-UK](https://github.com/PSLmodels/openfisca-uk).
+mircodf provides a number of useful tools for use with economic survey data.
+OpenFisca-UK builds off the [OpenFisca](https://openfisca.org/en/) platform, offering a microsimulation models for tax and benefits programs in the UK.
+
+In addition, four new models were added to the Library as incubating projects.  The [ui-calculator](https://github.com/PSLmodels/ui_calculator) model has recieved a lot of attention this year in the U.S., as it provides the capability to calculate unemployment insurance benefits across U.S. states, a major mode of delivering financial relief to individual during the COVID crisis. 
+[PCI-Outbreak](https://github.com/PSLmodels/PCI-Outbreak) directly relates to the COVID crisis, using maching learning and natural language processing to estimate the true degrees of COVID cases in China.
```
Comment (MaxGhenis) [inline _posts/2020-12-23-2020_year_in_review.md:21]:
```suggestion
[PCI-Outbreak](https://github.com/PSLmodels/PCI-Outbreak) directly relates to the COVID crisis, using machine learning and natural language processing to estimate the true degrees of COVID cases in China.
```

**Review by MaxGhenis — COMMENTED** [review]
thanks for kicking this off @jdebacker, fixed a couple typos

another achievement could be improving documentation across multiple models by moving to jupyter-book

**Comment by jdebacker** [comment]
@MaxGhenis I think this is complete.  Let me know if you have any suggestions to edit before merging.

**Comment by MaxGhenis** [comment]
LGTM! Merging.
image

**Pull request merged**